thirdparty/runtime_environments_test.go

```diff
@@ -21,3 +22,48 @@ func TestGetImageNames(t *testing.T) {
 	assert.NotEmpty(result)
 	assert.NotContains(result, "")
 }
+
+func TestGetPackages(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	assert := assert.New(t)
+	config := testutil.TestConfig()
+	testutil.ConfigureIntegrationTest(t, config, "TestGetPackages")
+	c := NewRuntimeEnvironmentsClient(config.RuntimeEnvironments.BaseURL, config.RuntimeEnvironments.APIKey)
+	manager := "pip"
+	ami := "ami-0e12ef25a5f7712a4"
+	opts := PackageFilterOptions{
+		Page:    0,
+		Limit:   10,
+		Manager: manager,
+	}
+	result, err := c.getPackages(ctx, ami, opts)
+	require.NoError(t, err)
+	assert.NotEmpty(result)
+	assert.Len(result, 10)
```

Comment:
I would require this length, or else 45 might panic. It also might be worth looping through and ensuring that manager matches for every result, not just the first one.